src/pkg/packager/create.go

```diff
@@ -8,51 +8,84 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"time"
 
-	"github.com/defenseunicorns/pkg/helpers/v2"
 	"github.com/zarf-dev/zarf/src/config"
 	"github.com/zarf-dev/zarf/src/pkg/layout"
+	"github.com/zarf-dev/zarf/src/pkg/logger"
 	"github.com/zarf-dev/zarf/src/pkg/message"
 	"github.com/zarf-dev/zarf/src/pkg/packager/creator"
+
+	"github.com/defenseunicorns/pkg/helpers/v2"
 )
 
 // Create generates a Zarf package tarball for a given PackageConfig and optional base directory.
 func (p *Packager) Create(ctx context.Context) error {
+	l := logger.From(ctx)
+	l.Info("starting package create")
+	cStart := time.Now()
+
+	// Begin setup
 	cwd, err := os.Getwd()
 	if err != nil {
 		return err
 	}
+	cfg := p.cfg
 
-	if err := os.Chdir(p.cfg.CreateOpts.BaseDir); err != nil {
-		return fmt.Errorf("unable to access directory %q: %w", p.cfg.CreateOpts.BaseDir, err)
+	// Set basedir
+	createOpts := cfg.CreateOpts
+	baseDir := createOpts.BaseDir
+	if err := os.Chdir(baseDir); err != nil {
+		return fmt.Errorf("unable to access directory %q: %w", baseDir, err)
 	}
-
+	// TODO(mkcp): Remove message on logger release
 	message.Note(fmt.Sprintf("Using build directory %s", p.cfg.CreateOpts.BaseDir))
+	l.Info("using build directory", "baseDir", baseDir)
 
-	pc := creator.NewPackageCreator(p.cfg.CreateOpts, cwd)
-
-	if err := helpers.CreatePathAndCopy(layout.ZarfYAML, p.layout.ZarfYAML); err != nil {
+	// Setup package creator
+	lo := p.layout
+	pc := creator.NewPackageCreator(createOpts, cwd)
+	if err := helpers.CreatePathAndCopy(layout.ZarfYAML, lo.ZarfYAML); err != nil {
 		return err
 	}
 
-	pkg, warnings, err := pc.LoadPackageDefinition(ctx, p.layout)
+	// Load package def
+	pkg, warnings, err := pc.LoadPackageDefinition(p.ctx, lo)
 	if err != nil {
 		return err
 	}
+	//  Store on packager config
 	p.cfg.Pkg = pkg
+	if len(warnings) > 0 {
+		l.Warn("warnings found when loading package definition", "warnings", warnings)
+	}
```

Comment:
```suggestion
	for _, warning := range warnings {
		l.Warn(warning)
	}
```
Thoughts on changing it to this? IMO it will read better

Reply:
Yeah definitely better with more lines